1_fetch/src/fetch_data.R

```diff
@@ -0,0 +1,25 @@
+# Load necessary packages
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+# Function to fetch data
+fetch_data <- function(output_path) {
+  # define file object
+  mendota_file <- file.path(output_path, 'model_RMSEs.csv')
+  # fetch data and overwrite file
+  item_file_download('5d925066e4b0c4f70d0d0599', names = 'me_RMSE.csv', destinations = mendota_file, overwrite_file = TRUE)
+  # return file
+  return(mendota_file)
+}
+
+
+##### Fetch Data #####
+
+# Set workspace parameters
+project_output_dir <- '1_fetch/out'
+
+# Call function
+data <- fetch_data(project_output_dir)
```

Comment:
Lines 19-25 are excellent info, but to move toward what we'll be asking of you in the next issue, you should extract these lines into another file - I used a `README.md` when I ran through this course myself - because soon we'll want this script to _only_ include function definitions and not any code that actually gets run to do stuff. Similar lines from the other `.../src/...R` scripts can go in that same `README.md`.

Reply:
One thought on `data <- fetch_data`: `data` is a very generic object name. It works for this simple analysis, but even something a little more descriptive like `eval_data` or `model_rmses` can be helpful as soon as an analysis gets a bit more complex.